packages/boba/api/alchemy-pay-api/package.json

```diff
@@ -10,6 +10,7 @@
   },
   "devDependencies": {
     "serverless": "^4.17.2",
+    "serverless-dotenv-plugin": "^6.0.0",
```

Comment:
This is probably fine for now, while doing manual deployments.  Ultimately though, we should probably be utilizing secrets manager for these.